### ISSUE TITLE
[1.8.22] iPhone 6 / iOS 9.3.2 Black frames when using Motifs 

### ISSUE THREAD

**Issue by xiuie** [Lomotif/lomotif-ios-issues#435]
Frequency - Sometimes

Steps to reproduce -
1. Launch app
2. Click on Start 
3. Go to Motif
4. Search "hug"
5. Add Motifs from the search (in particular the "Superhero family hug, and Frozen couple lift)
6. Click on apply on top right corner

Actual Result - Back frames in clip

Expected Result - Clips are just shorter

**Comment by tsukisa** [comment]
investigated, not iOS app issue.

**Comment by hiteshbheda** [comment]
@xiuie @tsukisa @tsukisa I can reproduce this issue for few motifs for build [1.8.23] on iPhone 6 / iOS 9.3.2

@xiuie I think it's Similar issue to #387

**Comment by vysohanych** [comment]
I can't reproduce it on build 1.8.24 on iPad mini iOS 8.3.

**Comment by pallavipillai** [comment]
The black frames are also visible when we select last motif from Nature section. 
Tested on build 1.8.29 on iPhone 6/iOS 9.3.2

![black](https://cloud.githubusercontent.com/assets/18537545/15631787/8e72389e-2598-11e6-8ad5-476f3521be29.PNG)

**Comment by hiteshbheda** [comment]
@pallavipillai Yes, I can also reproduce this black frame for that specific motif for build [1.8.29] on iPhone 6 / iOS 9.3.2
